src/Language/DifferentialDatalog/Compile.hs

```diff
@@ -166,6 +166,8 @@ rustLibFiles specname =
         , (dir </> "ovsdb/Cargo.toml"                   , $(embedFile "rust/template/ovsdb/Cargo.toml"))
         , (dir </> "ovsdb/lib.rs"                       , $(embedFile "rust/template/ovsdb/lib.rs"))
         , (dir </> "ovsdb/test.rs"                      , $(embedFile "rust/template/ovsdb/test.rs"))
+        , (dir </> "tcp_channel/Cargo.toml"             , $(embedFile "rust/template/tcp_channel/Cargo.toml"))
```

Comment:
do we need this in every project?
maybe we should add a --feature-like flag. Then we can probably remove ovsdb too.

Reply:
We should not need it everywhere. The question is whether it's worth adding some form of gate for it. Let me open an issue to discuss that.